metadata/src/test/java/org/apache/kafka/controller/QuorumControllerTest.java

```diff
@@ -394,7 +394,7 @@ public void testUncleanShutdownBroker() throws Throwable {
                     new BrokerRegistrationRequestData().
                         setBrokerId(brokerId).
                         setClusterId(active.clusterId()).
-                        setFeatures(brokerFeatures(MetadataVersion.IBP_3_0_IV1, MetadataVersion.IBP_3_8_IV0)).
+                        setFeatures(brokerFeatures(MetadataVersion.IBP_3_0_IV1, MetadataVersion.IBP_3_8_IV1)).
```

Comment:
Hmm, this is testing ELR. So it seems that it needs to be IBP_4_0_IV0. Ditto below. @CalvinConfluent : Could you confirm this?

Reply:
Yes, it requires IBP_4_0_IV0